### PR TITLE
feat: add snackbar and navigation component

### PR DESCRIPTION
**Pull request by Sahil25061999**
Added:
-navigation
-snackbar

### CODE REVIEW

**Comment by varun-Tad** [comment]
The code looks good to me. good to merge

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
Capture
Your input and search button are not vertically aligned. Button is higher than input.
You can give parent div display: flex & align-items: center.

Rest is looking fine 👍

**Comment by Rishabh-078** [comment]
The code is looking good, everything seems fine.
Just a suggestion: in the snack bar, the shadow can be reduced as it is not looking good.

File: component/snackbar/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  box-shadow: 4px 8px 40px var(--secondary-color);
+}
+
+.snackbar-secondary .btn {
+  background: var(--secondary-color);
+}
+
+.snackbar-accent {
+  background: var(--accent-color);
+  box-shadow: 4px 8px 40px var(--accent-color);
```
Comment (Rishabh-078) [inline component/snackbar/snackbar.css]: Shadow can be reduced

**Review by Rishabh-078 — CHANGES_REQUESTED** [review]

**Pull request merged**